### PR TITLE
Portuguese translation update

### PR DESCRIPTION
**Pull request by JLammeer**
Make sure to tell me, if there is anything else left for translating, ok?

### CODE REVIEW

**Comment by JLammeer** [comment]
Hell, I'm noob! =D
What did I do wrong?

--

wtf

**Comment by HugoGiraudel** [comment]
Hello @JLammeer! Thank you so much for taking the time. That’s fantastic. I’ll try to get around this over the week-end. :)

**Comment by JLammeer** [comment]
@morkro

Fixed the first problem.

However, still there is another one, wich makes me confused cause I've reproduced every missing commit.

File: pages/pt/_author.md [hunk 1]
```diff
@@ -1,15 +1,8 @@
 
 # Sobre o autor
 
-O meu nome é [Hugo Giraudel](http://hugogiraudel.com) e sou um Programador Front End francês em Berlim (Alemanha), desde 2015. Atualmente, trabalhando na [Edenspiekermann](http://edenspiekermann.com).
+O meu nome é [Hugo Giraudel](http://hugogiraudel.com) e sou um programador francês Front End de 25 anos, em Berlim (Alemanha), desde 2015. Atualmente, trabalhando na [N26](http://edenspiekermann.com).
```
Comment (morkro) [inline pages/pt/_author.md]: The URL doesn't match with the company :) It should be `https://next.n26.com`.

**Comment by morkro** [comment]
As afar as I can tell, that's because the links to your categories don't match anymore :) Since your headlines are now Portuguese you also have to update the hash reference.

```sh
- _site/pt/index.html
  *  linking to internal hash #calculations that does not exist (line 1635)
  *  linking to internal hash #color-formats that does not exist (line 1635)
  *  linking to internal hash #commenting that does not exist (line 1635)
  *  linking to internal hash #declaration-sorting that does not exist (line 1635)
  *  linking to internal hash #default-flag that does not exist (line 1635)
  *  linking to internal hash #documentation that does not exist (line 1635)
  *  linking to internal hash #encoding that does not exist (line 1635)
...
```

**Comment by JLammeer** [comment]
Certainly, more problems will appear, so I will try to redo it in the right way.

**Pull request closed without merging**